charts/enterprise/clusterissuer/questions.yaml

```diff
@@ -297,7 +297,7 @@ questions:
                         default: "my-selfsigned-ca"
                     - variable: crt
                       label: "Custom CA cert (experimental)"
-                      description: "certificate for Certiticate Authority"
+                      description: "CA Cert."
```

Comment:
please write CA in full

Reply:
Ive reverted questions.yaml as I'm no longer changing the user input to just take base64